internal/v1/api.yaml

```diff
@@ -1069,6 +1069,12 @@ components:
           $ref: '#/components/schemas/Customizations'
         metadata:
           $ref: '#/components/schemas/BlueprintMetadata'
+        custom_repositories_details:
+          type: string
+          additionalProperties: true
+          description: |
+            List of custom repositories including all the repository details needed in order
+            to recreate the repositories.
```

Comment:
Are you aiming for JSON in JSON string or arbitrary JSON object (without schema)? Because this looks like the former, from what I understand arbitrary key-value pairs are represented via

```
type: object
additionalProperties: true
```

Which should be more readable and easier to work with than JSON-encoded strings. Can you paste how such export looks like?

Reply:
Done, this is now an array of objects. :partying_face: